packages/near-fast-auth-signer/src/components/SignMultichain/evm/schema.ts

```diff
@@ -2,17 +2,15 @@ import * as yup from 'yup';
 
 import { BaseSendMultichainMessageSchema } from '../schema';
 
-export const SendEVMMultichainMessageSchema =  BaseSendMultichainMessageSchema.shape({
-  chain: yup
-    .number()
-    .required()
-    .test('is-evm', 'Invalid EVM chain value', (value) => value === 60),
-  chainId: yup.lazy((value) => yup.mixed<bigint>().test(
-    'is-valid-chainId',
-    `Invalid chainId value: ${value}`,
-    (val) => [BigInt(1), BigInt(56), BigInt(97), BigInt(11155111)].includes(val)
-  )),
-  maxFeePerGas:         yup.mixed<bigint>().optional(),
-  maxPriorityFeePerGas: yup.mixed<bigint>().optional(),
-  gasLimit:             yup.mixed<bigint>().optional(),
+export const SendEVMMultichainMessageSchema = BaseSendMultichainMessageSchema.shape({
```

Comment:
We should also validate the `chainId` like before. Also are we no longer supporting dapps specifying the gas themselves and only fetching when it's not specified?

Reply:
It's being validate on the derivationPath

Reply:
What about EVM chain previously validated here:
```typescript
chainId: yup.lazy((value) => yup.mixed<bigint>().test(
    'is-valid-chainId',
    `Invalid chainId value: ${value}`,
    (val) => [BigInt(1), BigInt(56), BigInt(97),
```